metric/example_test.go

```diff
@@ -142,6 +142,24 @@ func ExampleMeter_upDownCounter() {
 	}
 }
 
+// Gauges can be used to record non-additive values when changes occur.
+//
+// Here's how you might report the current speed of a cpu fan.
+func ExampleMeter_gauge() {
+	speedGauge, err := meter.Int64Gauge(
+		"cpu.fan.speed",
+		metric.WithDescription("Speed of CPU fan"),
+		metric.WithUnit("RPM"),
+	)
+	if err != nil {
+		panic(err)
+	}
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		// hard-code 1500 RPM for demonstrative purposes
+		speedGauge.Record(r.Context(), 1500)
+	})
```

Comment:
An example where the fan speed is recorded during an HTTP request seems very artificial. Why not use something more realistic? Maybe one of the examples [mentioned in the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#gauge):

- subscribe to change events for the background noise level
- subscribe to change events for the CPU fan speed

Reply:
Thank you for the feedback. I will update the example as suggested.

Reply:
I updated the example to be like subscribing to fan speed change.